FontMenuNameDB

```diff
@@ -7,146 +7,79 @@
 
 # Variable fonts -----------------------------
 
-[SourceSerifVariable-Roman]
-    f=Source Serif Variable
+[SourceSerif4Variable-Roman]
+    f=Source Serif 4 Variable
     s=Roman
 
-[SourceSerifVariable-Italic]
-    f=Source Serif Variable
+[SourceSerif4Variable-Italic]
+    f=Source Serif 4 Variable
     s=Italic
 
 # Masters --------------------------------------
 
-[SourceSerifMaster-0]
-    f=Source Serif Master
-    s=0
-	l=Source Serif Master 0
-
-[SourceSerifMaster-1]
-    f=Source Serif Master
-    s=1
-    l=Source Serif Master 1
-
-[SourceSerifMaster-2]
-    f=Source Serif Master
-    s=2
-    l=Source Serif Master 1
-
-[SourceSerifMaster-3]
-    f=Source Serif Master
-    s=3
-
-
-[SourceSerifMaster-Italic0]
-    f=Source Serif Master
-    s=Italic 0
-    l=Source Serif Master 0
-
-[SourceSerifMaster-Italic1]
-    f=Source Serif Master
-    s=Italic 1
-    l=Source Serif Master 1
-
-[SourceSerifMaster-Italic2]
-    f=Source Serif Master
-    s=Italic 2
-    l=Source Serif Master 1
-
-
-[SourceSerifMaster-Caption0]
-    f=Source Serif Master
+[SourceSerif4Variable-Caption0]
+    f=Source Serif 4 Variable
     s=Caption 0
-    l=Source Serif Master Caption 0
```

Comment:
I am not in favor of taking the Masters out of the FontMenuNameDB. I build the masters regularly while testing fonts, and I don’t want them to be called “variable” (for the simple reason that they are not variable). I understand they need to get their family name from somewhere once the VF is built, but I’d love to find a more flexible solution for this.

Reply:
I found a solution for this.
For example, 
- familyName in UFO `Source Serif Master`
- styleName in UFO `Text 0`
- postscriptFontName in UFO `SourceSerif4Variable-Text0`

This allows me to generate a style name for static masters on the fly, and builds properly-named master OTFs.